src/read.c

```diff
@@ -3818,13 +3818,15 @@ static avifResult avifParseMinimizedImageBox(avifMeta * meta, uint64_t rawOffset
     // Chunks
     avifCodecConfigurationBox alphaItemCodecConfig = { 0 };
     if (hasAlpha && alphaItemDataSize != 0 && alphaItemCodecConfigSize != 0) {
+        // 'av1C' always uses 4 bytes.
```

Comment:
I copied this comment from src/write.c. It took me a while to figure out we why are comparing the codec config size with 4.